libraries/I2S/src/I2S.cpp

```diff
@@ -188,6 +240,11 @@ bool I2S::begin() {
 }
 
 void I2S::end() {
+    if (_MCLKenabled && _running) {
+        pio_sm_set_enabled(_pioMCLK, _smMCLK, false);
+        _MCLKenabled = false;
```

Comment:
Why disable the mclk on ::end?  We don't change any other settings on ::end so I would say we should let the value stick as-it-was.